simtools/schemas/model_parameters/nsb_reference_spectrum.schema.yml

```diff
@@ -0,0 +1,34 @@
+%YAML 1.2
+---
+title: Schema for nsb_reference_spectrum model parameter
+version: 0.1.0
+meta_schema: simpipe-schema
+meta_schema_url: https://raw.githubusercontent.com/gammasim/simtools/main/simtools/schemas/model_parameter_and_data_schema.metaschema.yml
+meta_schema_version: 0.1.0
+developer_note: To be replaced by a data table
+name: nsb_reference_spectrum
+description: |-
+  Nightsky spectrum from Benn & Ellison (astro-ph/9909153 actually),
```

Comment:
I would remove the 'actually', as it doesn't really have meaning (without a Benn & Elisson reference), but I don't want to be pedantic!

Reply:
This was copy pasted directly from the spectrum file from KB, but I removed it, no problem.